src/hoi4_world/countries/hoi4_country.cpp

```diff
@@ -0,0 +1,7 @@
+#include "hoi4_country.h"
+
+bool hoi4::Country::AreElectionsAllowed() const
+{
+   // Strictly from V3 or ideology mapping dependent?
```

Comment:
This comment can be deleted. Feel free to open an issue if you've got other ideas in mind.